DOCUMENTATION.md

```diff
@@ -1,16 +1,19 @@
 ## Version: 2.11.0
 
-The SAP Cloud SDK for JavaScript and TypeScript consists of multiple libraries to enable you to write side-by-side extensions on SAP Business Technology Platform.
```

Comment:
technically our product name is "SAP Cloud SDK for JavaScript"

do we need to disambiguate that it can also be used with typescript? does it make sense to do that in another sentence?

> The SAP Cloud SDK for JavaScript consists of multiple libraries to enable you to write side-by-side extensions on SAP Business Technology Platform.
> It works great with both JavaScript and TypeScript.

Reply:
I agree with this